src/h5web/toolbar/controls/DomainSlider/DomainSlider.tsx

```diff
@@ -36,8 +36,8 @@ function DomainSlider(props: Props) {
     setSliderDomain(visDomain);
   }, [visDomain, setSliderDomain]);
 
-  const isAutoMin = customDomain[0] === undefined;
-  const isAutoMax = customDomain[1] === undefined;
+  const isAutoMin = customDomain[0] === null;
+  const isAutoMax = customDomain[1] === null;
```

Comment:
I've added a comment to the model, but FYI, `undefined` gets persisted as `null` to `localStorage` (because of `JSON.stringify`), so changing from `undefined to `null` everywhere, means we don't have to change it dynamically in zustand's `persist` middleware.